File: Helicoptero.cpp
// Archivo Helicoptero.cpp
#include "Helicoptero.h"

// Implementación de los métodos de la clase Helicoptero
Helicoptero::Helicoptero(string tipo, string marca, string modelo, int Num_Asientos, int velo_max, int autonomia, int agno, string estado,  int no_rotores, int capa_elevacion, string uso)
    : Vehiculos_Aereos(tipo, marca, modelo, Num_Asientos, velo_max, autonomia, agno, estado), No_Rotores(no_rotores), Capa_Elevacion(capa_elevacion), uso(uso) {}

int Helicoptero::getNo_Rotores() const {
    return No_Rotores;
}

void Helicoptero::setNo_Rotores(int No_Roto) {
    No_Rotores = No_Roto;
}

int Helicoptero::getCapa_Elevacion() const {
    return Capa_Elevacion;
}

void Helicoptero::setCapa_Elevacion(int New_Ele) {
    Capa_Elevacion = New_Ele;
}

void Helicoptero::Show_Uso() {
    cout << uso << endl;
}

void Helicoptero::Show_Info_H(){
	Show_Info();
	cout << "Numero de Rotores: " << No_Rotores << endl;
	cout << "Capacidad de Elevacion: " << Capa_Elevacion << endl;
	cout << "Uso especifico: " << uso << endl;
	cout << endl;
}

File: proyectoAeropuertoPython/helicoptero.py
from VehiculosAereos import VehiculosAereos  # Importa la clase Vehiculos_Aereos desde el archivo correspondiente

class Helicoptero(VehiculosAereos):
    def __init__(self, id,tipo, marca, modelo, num_asientos, velo_max, autonomia, agno, estado, no_rotores, capa_elevacion, uso):
        super().__init__(id,tipo, marca, modelo, num_asientos, velo_max, autonomia, agno, estado)
        self.No_Rotores = no_rotores
        self.Capa_Elevacion = capa_elevacion
        self.uso = uso
        self.Puerta="sin asignar"

    def getNo_Rotores(self):
        return self.No_Rotores

    def setNo_Rotores(self, No_Roto):
        self.No_Rotores = No_Roto

    def getCapa_Elevacion(self):
        return self.Capa_Elevacion

    def setCapa_Elevacion(self, New_Ele):
        self.Capa_Elevacion = New_Ele
    def get_Uso(self):
        return self.uso

    def Show_Uso(self):
        print(self.uso)

    def Show_Info_H(self):
        self.Show_Info()
        print("Numero de Rotores:", self.No_Rotores)
        print("Capacidad de Elevación:", self.Capa_Elevacion)
        print("Uso específico:", self.uso)
